keg_elements/db/mixins.py

```diff
@@ -389,3 +390,48 @@ def delete_cascaded(cls):
 
 sa.event.listen(SoftDeleteMixin, 'before_delete', SoftDeleteMixin.sqla_before_delete_event,
                 propagate=True)
+
+
+class LookupMixin(DefaultMixin):
```

Comment:
Let's base this on SoftDeleteMixin, so we get all of the advantages there. E.g. "delete" action sets the timestamp. Including DefaultMixin feels too monolithic. I think we can assume that whatever uses this mixin will have the methods it calls.

We should also have a nullable `code` or `key` field, which will allow lookup records to be fetched in code without reference to a changeable label.